lib/check_reqs.js

```diff
@@ -110,7 +110,9 @@ module.exports.get_gradle_wrapper = function () {
     let program_dir;
     // OK, This hack only works on Windows, not on Mac OS or Linux.  We will be deleting this eventually!
     if (module.exports.isWindows()) {
-        const result = execa.sync(path.join(__dirname, 'getASPath.bat'));
+        // "spawn" option enabled for CVE-2024-27980 (Windows) Mitigation
```

Comment:
```suggestion
        // "shell" option enabled for CVE-2024-27980 (Windows) Mitigation
```

I wrote in "spawn" because that was in the original PR but since we determine the "shell" option is correct, that should be reflected in the comment too.